src/makeRobotAccountant.js

```diff
@@ -7,6 +7,25 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      if (count <= 3) {
+        return a + b;
+      }
+
+      if (count >= 4 && (count % 2 === 0)) {
+        return `Bzzz... Error!`;
+      }
+
+      if (count >= 4 && (count % 2 !== 0)) {
+        return a + b;
+      }
```

Comment:
```suggestion
      return a + b;
```
Если дошло до этого участка кода - значит ни один из предыдущих if-ов не выполнился, и уже можно ничего не проверять